src/monorepo-npm-publisher/package.json

```diff
@@ -8,7 +8,7 @@
   "main": "src/index",
   "sideEffects": false,
   "dependencies": {
-    "@adeira/babel-preset-adeira": "^1.2.0",
+    "@adeira/babel-preset-adeira": "^2.0.0",
```

Comment:
I was thinking: could this have some undesired implications on our released NPM packages? Like different React version requirements for example? Should we do it more carefully in such a case?

Reply:
I don't think so, or maybe there is something I am not seeing. Would my mind specify exactly what you fear might happen?

Reply:
Ok, I see what you mean now, and we already created this problem for ourselves, since I forgot to set `reactRuntime: 'automatic'` in `monorepo-npm-publisher`. 

We definitely need to find some solution here before we release a new version of this package.

However, this MR is not releasing this package.

Reply:
I made a note of it in the changelog, under the unreleased section.

Reply:
@tbergq So from now on the NPM publisher will release React code which is only compatible with the new JSX transform, do I understand it correctly? So every package will require peer React dependency at least 16.4, right?

Reply:
> So from now on the NPM publisher will release React code which is only compatible with the new JSX transform, do I understand it correctly?

Yes, but I think we could add an option to which runtime to target. 

> So every package will require peer React dependency at least 16.4, right?

Well, `16.14.0`

Reply:
Ok, I added the option to configure `reactRuntime` for `monorepo-npm-publisher`.